examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -200,8 +199,36 @@ class WebGPURenderer {
 
 		}
 
+		// feature support
+
+		const features = [
+			GPUFeatureName.DepthClipControl,
+			GPUFeatureName.Depth32FloatStencil8,
+			GPUFeatureName.TextureCompressionBC,
+			GPUFeatureName.TextureCompressionETC2,
+			GPUFeatureName.TextureCompressionASTC,
+			GPUFeatureName.TimestampQuery,
+			GPUFeatureName.IndirectFirstInstance,
+			GPUFeatureName.ShaderF16,
+			GPUFeatureName.RG11B10UFloat,
+			GPUFeatureName.BGRA8UNormStorage,
+			GPUFeatureName.Float32Filterable
+		];
+
+		const supportedFeatures = [];
+
+		for ( const name of features ) {
+
+			if ( adapter.features.has( name ) ) {
+
+				supportedFeatures.push( name );
+
+			}
+
+		}
+
 		const deviceDescriptor = {
-			requiredFeatures: parameters.requiredFeatures,
+			requiredFeatures: supportedFeatures,
```

Comment:
Could it be, that, for example, some features conflict with each other? Or the renderer just, for example, becomes slower when one of them is enabled?

Reply:
I'm not aware of any of that. 

But a review from one of the WebGPU spec maintainers would be of course best.